builder/bundles.go

```diff
@@ -877,7 +878,40 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(filename, version string) error {
+func fixOSRelease(b *Builder, filename, version string) error {
+
+	//Replace the default os-release file if customized os-release file found
+	path := b.Config.Mixer.OSReleasePath
+	if path != "" {
+
+		// Validate file path
+		_, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
+
+		fSrc, err := os.OpenFile(path, os.O_RDONLY, 0666)
+		if err != nil {
+			return err
+		}
```

Comment:
Use `defer fSrc.Close()` here to prevent file file descriptor leak.

Reply:
I tried to defer the file close() operation but i always get empty fill for /usr/lib/os-release file after mixer build bundle.
I suspect the copy operation has not been completed at the moment when file opened for VERSION_ID. Thus, I did a manual file close before same file being opened again. 
However, this will be be issue anymore with your suggestion to open different file instead of copy.